include/Page/Base.php

```diff
@@ -232,19 +233,20 @@ public function enqueue_script() {
 			'select_one' => __( 'Please select least one option', 'bulk-move' ),
 		);
 
-		// TODO: Add this from Bulk_Move_Posts class using a filter.
-		$bulk_move_posts = array(
-			'action_get_taxonomy' => 'load_custom_taxonomy_by_post_type',
-			'action_get_terms'    => 'load_custom_terms_by_taxonomy',
-//			'nonce'               => wp_create_nonce( self::BOX_CUSTOM_TERMS_NONCE ),
-		);
-
 		$translation_array = array(
-			'msg'             => $msg,
-			'error'           => $error,
-			'bulk_move_posts' => $bulk_move_posts,
+			'msg'   => $msg,
+			'error' => $error,
 		);
 
+		/**
+		 * Filters the localized translations for bulk-move.js.
+		 *
+		 * @since 2.0.0
+		 *
+		 * @param array $translation_array A single dimensional translation array.
+		 */
+		$translation_array = apply_filters( 'bm_bulk-move_js_i18n', $translation_array );
```

Comment:
Let's use the name `bm_js_translations` as the name of the filter.

Reply:
> Let's use the name bm_js_translations as the name of the filter.

Fixed and I sticked with the existing name that's used throughout the plugin.